core/domain/classifier_services.py

```diff
@@ -469,21 +619,27 @@ def create_classifier_training_job_for_reverted_exploration(
         exploration: Exploration. Exploration domain object.
         exploration_to_revert_to: Exploration. Exploration to revert to.
     """
-    classifier_training_jobs_for_old_version = get_classifier_training_jobs(
-        exploration.id, exploration_to_revert_to.version,
-        list(exploration_to_revert_to.states.keys()))
-    job_exploration_mappings = []
+    classifier_training_job_maps_for_old_version = (
```

Comment:
What is the difference between "maps" (here) and "mappings" (4 lines below)? If they mean the same thing, use the same word.

Reply:
Its same. Done